internal/client/client_batch.go

```diff
@@ -579,7 +586,6 @@ func (c *batchCommandsClient) send(forwardedHost string, req *tikvpb.BatchComman
 			zap.String("forwardedHost", forwardedHost),
 			zap.Error(err),
 		)
-		c.failPendingRequests(err)
```

Comment:
This is introduced by https://github.com/pingcap/tidb/pull/12819 , need to do more test about this to avoid regression.

Reply:
I got it, and already add test for it.